src/test/java/net/openhft/chronicle/bytes/BytesTest.java

```diff
@@ -45,6 +45,7 @@
 import static org.junit.Assert.*;
 import static org.junit.Assume.assumeFalse;
 import static org.junit.Assume.assumeTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
I did not convert the test to JUnit 5 - just did the smallest thing that could work